src/cr/cube/min_base_size_mask.py

```diff
@@ -4,10 +4,7 @@
 
 from __future__ import division
 
-import numpy as np
-
 from cr.cube.util import lazyproperty
-from cr.cube.enums import DIMENSION_TYPE as DT
 
 
 class MinBaseSizeMask(object):
```

Comment:
Is there a plan for this module to go away and it's contents to be consolidated into measures and wherever?

Reply:
Sure, it can. There's a lot of boiler plate consolidation like that in my last (current) PR, so I can add it to that.